.github/workflows/ci.yml

```diff
@@ -19,4 +25,4 @@ jobs:
         run: flake8 .
```

Comment:
I think it's probably better to have flake8 and pytest in separate builds. Doesn't make so much sense to test flake8 in different Python versions.

Reply:
Thank you for the recommendations, I’m going to implement it.